app-editors/okteta/okteta-0.26.13-r1.ebuild

```diff
@@ -21,7 +21,7 @@ fi
 
 LICENSE="GPL-2 handbook? ( FDL-1.2 )"
 SLOT="5"
-IUSE="crypt"
+IUSE="crypt handbook kparts"
```

Comment:
`handbook`: unnecessary, see ECM_HANDBOOK meta var.
`kparts`: what do we gain by making this optional? will users know what to make of that USE flag name?

Reply:
> It does add the flag there, but it seemingly does not work for some reason

I can definitely see it here, so...?
`[ebuild   R    ] app-editors/okteta-0.26.13:5::gentoo  USE="crypt handbook -debug -designer -test" 0 KiB`

Reply:
I'm wondering why this hasn't been reported as bug in 4 years...

Reply:
> I guess noone bothers by unmerging all build deps every time and checking how the package builds from the scratch, as I do 🙂

Well, tinderboxes do.

Reply:
The issue is you can't _just_ unmerge them if anything else RDEPENDs on those which happen to be a BDEPEND for this package. But yeah, tinderboxes tend to catch most of this.